media-video/noad/noad-0.8.8.ebuild

```
@@ -0,0 +1,69 @@
+# Copyright 2021-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Mark commercial breaks in VDR recordings"
+HOMEPAGE="https://github.com/madmartin/noad"
+SRC_URI="https://github.com/madmartin/noad/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+ffmpeg imagemagick libmpeg2"
+REQUIRED_USE="|| ( ffmpeg libmpeg2 )"
+
+DEPEND="
+	libmpeg2? ( media-libs/libmpeg2:= )
+	ffmpeg? ( media-video/ffmpeg )
+	imagemagick? ( media-gfx/imagemagick )"
+RDEPEND="${DEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		$(usex imagemagick '--with-magick') \
```

This particular line is wrong.  Look at the generated configure call (I shortened it a little):
`./configure --prefix=/usr [...] --libdir=/usr/lib64 no --without-libmpeg2 --with-tools`
That "no" is the default output of `usex` if imagemagick is disabled.  Fortunately autoconf ignores it :)
In EAPI 8 you can use [usev](https://mgorny.pl/articles/the-ultimate-guide-to-eapi-8.html#usev-now-accepts-a-second-argument) instead.

```suggestion
		$(usev imagemagick --with-magick) \
```